idea/data-storage/src/common/helpers/get-hex-wasm-state.ts

```diff
@@ -0,0 +1,10 @@
+import { CreateType, Hex } from '@gear-js/api';
+import * as dotenv from 'dotenv';
+
+dotenv.config();
+
+export function getHexWasmState(wasmStateBuff: Buffer): Hex {
+  if(process.env.TEST_ENV_UNIT) return 'hex' as Hex;
+
+  return CreateType.create('Bytes', wasmStateBuff).toHex();
+}
```

Comment:
It much better to compare wasms by their's hash, not full wasm in hex representaion. For this purpose you can use `generateCodeHash` function from @gear-js/api